http/src/handler.rs

```diff
@@ -33,6 +36,7 @@ impl<M: Metadata, S: Middleware<M>> ServerHandler<M, S> {
 		jsonrpc_handler: Rpc<M, S>,
 		cors_domains: CorsDomains,
 		cors_max_age: Option<u32>,
+		allowed_headers: AccessControlAllowHeaders,
```

Comment:
Please refer to the type in a consistent way (either `cors::AccessControlAllowHeaders` or directly). TBH I prefer the former.